domain/tokens.go

```diff
@@ -6,8 +6,12 @@ import (
 
 // Token represents the token's domain model
 type Token struct {
+	// Name
+	Name string `json:"name"`
 	// HumanDenom is the human readable denom.
 	HumanDenom string `json:"symbol"`
+	// Denom is the chain denom of the token.
+	Denom string `json:"coinMinimalDenom"`
```

Comment:
Nit: maybe it would make field more clear to name as `CoinMinimalDenom` as well? Can we add examples for both this new field and `HumanDenom`?

Reply:
Renamed the variable in the latest commit for consistency.